apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -1,67 +1,99 @@
-<div class="container-lg px-4 lg:mx-auto">
-  <p
-    class="font-title text-[28px] font-medium mt-8 mb-5 text-title text-center sm:mt-12 sm:mb-[-22px] sm:text-left"
-    translate
+<ng-container *ngIf="apiLinks$ | async as apiLinks">
+  <div
+    class="flex flex-row gap-x-4 items-center justify-center md:justify-start"
   >
-    record.metadata.api
-  </p>
+    <p
+      class="font-title text-[28px] font-medium text-title text-center sm:text-left"
+      translate
+    >
+      record.metadata.api
+    </p>
+    <div class="flex flex-row gap-x-4 items-center" *ngIf="hasSteps">
+      <gn-ui-button
+        [style.--gn-ui-button-rounded]="'100%'"
+        [style.--gn-ui-button-width]="'8px'"
+        [style.--gn-ui-button-height]="'8px'"
+        [style.--gn-ui-button-padding]="'12px'"
+        [type]="isFirstStep ? 'default' : 'outline'"
+        [disabled]="isFirstStep"
+        (buttonClick)="slideToPrevious()"
+      >
+        <mat-icon
+          class="material-symbols-outlined text-[14px] text-center pt-[5px]"
+        >
+          arrow_back
+        </mat-icon>
+      </gn-ui-button>
+      <gn-ui-button
+        [style.--gn-ui-button-rounded]="'100%'"
+        [style.--gn-ui-button-width]="'8px'"
+        [style.--gn-ui-button-height]="'8px'"
+        [style.--gn-ui-button-padding]="'12px'"
+        [type]="isLastStep ? 'default' : 'outline'"
+        [disabled]="isLastStep"
+        (buttonClick)="slideToNext()"
+      >
+        <mat-icon
+          class="material-symbols-outlined text-[14px] text-center pt-[5px]"
+        >
+          arrow_forward
+        </mat-icon>
+      </gn-ui-button>
+    </div>
+  </div>
   <gn-ui-carousel
+    [itemsLength]="apiLinks.length"
```

Comment:
this is not needed anymore I think
(sorry I posted that on the wrong line initially!)